Apps/AdminWebClient/src/ClientApp/src/services/httpDelegate.ts

```diff
@@ -128,17 +128,6 @@ export default class HttpDelegate implements IHttpDelegate {
                     console.log(errorMessage);
                     return reject(errorMessage);
                 });
-            // TODO: Axios has bug with the delete method not using data fields.
```

Comment:
we need to keep this todo until it gets fixed

Reply:
bug created https://qslvic.visualstudio.com/Health%20Gateway/_workitems/edit/10506